tests/test_dmm.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2020 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import cast
+from unittest.mock import patch
+
+import numpy as np
+import pytest
+
+from pulser.channels.dmm import DMM
+from pulser.register.base_register import BaseRegister
+from pulser.register.mappable_reg import MappableRegister
+from pulser.register.register_layout import RegisterLayout
+from pulser.register.weight_maps import DetuningMap
+
+
+@pytest.fixture
+def trap_coordinates():
```

Comment:
It seems like you're not using this outside of `layout`, so maybe it's not worth it to make it a fixture